R/crate.R

```diff
@@ -73,7 +78,7 @@ NULL
 #' # explicitly set its environment to the crate environment with the
 #' # set_env() function from rlang:
 #' crate(rlang::set_env(fn))
-crate <- function(.fn, ...) {
+crate <- function(.fn, ..., parent_env = baseenv()) {
```

Comment:
```suggestion
crate <- function(.fn, ..., .parent_env = baseenv()) {
```

Oh, we also dot prefix our argument names (like `.fn` there) when our functions take _named dots_, i.e. when you expect `...` to be name-value pairs. This decreases the chance of collision with some user provided name.